subset_vds.py

```diff
@@ -66,28 +73,28 @@ def check_output_already_exists(output_format: list[str], infile_name: str) -> N
             if to_path(
                 output_path(f"{infile_name}_subset", category="analysis")
             ).exists():
-                output_errors += f"The output VDS {infile_name}_subset.vds already exists. Refusing to overwrite it.\n"
+                output_errors += f"The output VDS {to_path(output_path(infile_name, category='analysis'))}_subset.vds already exists. Refusing to overwrite it.\n"
```

Comment:
it's not an expensive operation, but you're creating this Path object once to check it, and a second time to print it. You could create it as a variable instead of creating it twice.

Reply:
Hopefully this addresses it.